router/core/router.go

```diff
@@ -660,28 +680,28 @@ func (r *Router) newServer(ctx context.Context, routerConfig *nodev1.RouterConfi
 			subChiRouter.Use(traceHandler.Handler)
 		}
 
+		subChiRouter.Use(NewWebsocketMiddleware(rootContext, WebsocketMiddlewareOptions{
```

Comment:
In the last meeting, we decided to not call the module hooks unless they are working as intended. Currently, the OnOriginRequest is called for upgrade requests. OnOriginRequest should only be called for subgraph requests.

Reply:
Good catch. I've amended it to skip upgrade requests.